src/features/store/components/product-view.tsx

```diff
@@ -0,0 +1,230 @@
+              width={600}
+              height={600}
+              className="w-full h-auto object-contain"
+            />
+          </div>
+
+          {/* Thumbnails */}
+          <div className="grid grid-cols-4 gap-2">
+            {productImages.map((_, index) => (
+              <Button
+                key={index}
+                onClick={() => setSelectedImage(index)}
+                className={`bg-gray-100 rounded-lg overflow-hidden border-2 ${selectedImage === index ? "border-green-500" : "border-transparent"
+                  }`}
+              >
+                <img
+                  src={DeliveryBgImg || "/placeholder.svg"}
+                  alt={`Thumbnail ${index + 1}`}
+                  width={100}
+                  height={100}
+                  className="w-full h-auto object-contain"
+                />
+              </Button>
+            ))}
+          </div>
+        </div>
+
+        {/* Product Info */}
+        <div>
+          <h1 className="text-3xl font-bold mb-4">{productDetails?.name}</h1>
+
+          <div className="flex items-center mb-2">
+            <ShoppingCart className="h-5 w-5 text-gray-500 mr-2" />
+            <span className="text-gray-600">Sold by Loan Farmer Limited</span>
+            <span className="ml-4 text-green-600 text-sm font-medium">In Stock</span>
+          </div>
+
+          <div className="text-2xl font-bold mb-6">${productDetails?.price}</div>
+
+          <div className="border-t border-b border-gray-200 py-6 mb-6">
+            <p className="text-gray-700 mb-4">
+              {productDetails?.description}
+            </p>
+          </div>
+
+          {/* Quantity Selector */}
+          <div className="mb-6">
+            <label htmlFor="quantity" className="block text-sm font-medium text-gray-700 mb-2">
+              Quantity
+            </label>
+            <div className="flex items-center">
+              <Button onClick={decreaseQuantity} className="p-2 border border-gray-300 rounded-l-md hover:bg-gray-100">
+                <Minus className="h-4 w-4" />
+              </Button>
+              <input
+                type="text"
+                id="quantity"
+                value={quantity.toString().padStart(2, "0")}
+                readOnly
+                className="w-12 text-center border-t border-b border-gray-300 py-2"
+              />
+              <Button onClick={increaseQuantity} className="p-2 border border-gray-300 rounded-r-md hover:bg-gray-100">
+                <Plus className="h-4 w-4" />
+              </Button>
+
+              <Button className="ml-4 text-white py-2 px-6 rounded-md flex-grow md:flex-grow-0 md:ml-6">
+                Add to cart
+              </Button>
+            </div>
+          </div>
+
+          {/* Category */}
+          <div className="mb-4">
+            <span className="text-gray-600">Category: </span>
+            <Link to="/category/food" className="text-gray-800 hover:text-green-600">
```

Comment:
instead of hardcoding create the paths and use it. I see you have started using fetching data from this page. That is why i am asking if this are still temporary